backend/src/app/rpc/commands/auth.clj

```diff
@@ -275,7 +275,9 @@
                               :viewed-walkthrough? false
                               :nudge {:big 10 :small 1}
                               :v2-info-shown true
-                              :release-notes-viewed (:main cf/version)})
+                              :release-notes-viewed (:main cf/version)
+                              :newsletter-news? (or newsletter-news false)
+                              :newsletter-updates? (or newsletter-updates false)})
```

Comment:
This props already exists without `?` on the name, so I think we should respect the previous naming.